.github/meta_issue_template/release-1.1.1-plugin-issues.md

```diff
@@ -0,0 +1,35 @@
+Coming from [opensearch-build#870](https://github.com/opensearch-project/opensearch-build/issues/870), release version 1.1.1. Please follow the following checklist.
+### You can find all the date in above issue 
+### (We only make changes to OpenSearch-Dashboards to 1.1.1, therefore, OpenSearch stays 1.1.0.)
+
+### Preparation
+
+- [ ] Assign this issue to a release owner.
+- [ ] Finalize scope and feature set and update [the Public Roadmap](https://github.com/orgs/opensearch-project/projects/1).
+- [ ] Create, update, triage and label all features and issues targeted for this release with v1.1.1.
+
+### CI/CD
+
+- [ ] Increment version on main to `1.1.1.0`.
+- [ ] Ensure working and passing CI.
+- [ ] Re(add) this repo to the [manifest](https://github.com/opensearch-project/opensearch-build/blob/main/manifests/1.1.1).
+
+### Pre-Release
+
+- [ ] Branch and build from a `1.1` branch.
```

Comment:
This should say 1.1.1.

Reply:
So 1.1.1 branch has 3 digit because it is patch version?
I thought we always use 2 digits to contain both minor and patch releases.
Thanks.

Reply:
I think you're right, we never create a 1.1.1 branch, just fix it on 1.1, since there's no backporting to 1.1.1.

Reply:
Fixed.